CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 0.7.9dev
 
+* [Feature] Modified `histogram` command to support data with NULL values ([#176])
```

Comment:
I'll merge this now but the format is wrong @AnirudhVIyer, it should be: `(#NUMBER)`, not `([#NUMBER])`